src/main/java/com/google/sps/servlets/SeederServlet.java

```diff
@@ -1,11 +1,13 @@
 package com.google.sps.servlets;
 
-import com.google.sps.seeder.AppSeeder;
+
+import com.google.sps.seeder.*;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
```

Comment:
You'll also need to add your Seeder class to the doGet and call your seedDB function